t/bad_ancestry_error.t

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+# (the numbers are years)
+#
+# top:2019
+# ├─ bar:2018
+# │  └─ xyz:2010
+# └─ foo:2017
+#    └─ xyz:2011
```

Comment:
It would be helpful if the numbers after the `:` here corresponded to the numbers in the commit messages below, just for understanding the test